src/components/About/about.css

```diff
@@ -0,0 +1,106 @@
+.about-wrapper {
+  width: 340px;
+  margin: 0 auto;
+  border-radius: 20px;
+  background-color: var(--primary-element-color);
+}
+
+.about-title {
+  width: 300px;
+  display: block;
+  margin: 0 auto;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  color: var(--primary-text-color);
+}
+
+.about-text {
+  width: 300px;
+  height: auto;
+  margin: 0 auto;
```

Comment:
seems like this rule repeats many times. Maybe we can define a broblem here

Reply:
done